code/modules/organs/subtypes/parasite.dm

```diff
@@ -0,0 +1,176 @@
+/obj/item/organ/parasite
+	name = "parasite"
+	icon = 'icons/mob/alien.dmi'
+	icon_state = "burst_lie"
+	dead_icon = "bursted_lie"
+
+	organ_tag = "parasite"
+	var/stage = 1
+	var/max_stage = 4
+	var/stage_ticker = 0
+	var/stage_interval = 600 //time between stages, in seconds
+
+/obj/item/organ/parasite/process()
+	..()
+
+	if(!owner)
+		return
+
+	if(stage < max_stage)
+		stage_ticker += 2 //process ticks every ~2 seconds
+
+	if(stage_ticker >= stage*stage_interval)
+		stage = min(stage+1,max_stage)
+
+///////////////////
+///K'ois Mycosis///
+///////////////////
+
+/obj/item/organ/parasite/kois
+	name = "k'ois mycosis"
+	//icon_state = "kois-on"
+	//dead_icon = "kois-off"
+
+	organ_tag = "kois"
+
+	parent_organ = "chest"
+
+/obj/item/organ/parasite/kois/process()
+	..()
+
+	if(stage >= 1)
+		if(prob(5*stage))
+			owner << "<span class='warning'>You feel a stinging pain in your abdomen!</span>"
+			owner.emote("me",1,"winces slightly.")
+			owner.adjustHalLoss(5)
+
+		if(prob(10))
+			owner.emote("cough")
+
+		if(prob(5))
+			spawn owner.emote("me", 1, "coughs up blood!")
```

Comment:
I don't see why this needs to be spawned.

Reply:
Should investigate lungs in general then, as I just copy-pasted with the assumption that there's some arcane technical reason you would spawn an emote()